setup.py

```diff
@@ -181,6 +180,13 @@ def _patch_cfile(self, cfile):
     readme = f.read()
 
 
+with open(os.path.join(os.path.dirname(__file__), 'asyncpg', '__init__.py')) as f:
+    for line in f:
+        if line.startswith('__version__ ='):
+            _, _, version = line.partition('=')
+            VERSION = version.strip(" \n'\"")
+
```

Comment:
Must raise `RuntimeError` here if the version could not be parsed.

Reply:
Done